src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -1075,7 +1075,7 @@ await VerifyCachedDiagnosticsAsync(
             : root.DescendantNodes().OfType<CodeAnalysis.CSharp.Syntax.InvocationExpressionSyntax>().First().Span;
 
         await analyzerService.GetDiagnosticsForIdsAsync(
-            sourceDocument.Project.Solution, sourceDocument.Project.Id, sourceDocument.Id, diagnosticIds: null, shouldIncludeAnalyzer: null,
+            sourceDocument.Project, sourceDocument.Id, diagnosticIds: null, shouldIncludeAnalyzer: null,
```

Comment:
most callers had a project, but would then use it to pass a Solution+ProjectId.  Simpler to just pass Project